generator/python/resources/templates/example_statistics.mustache

```diff
@@ -1,9 +1,9 @@
 import re
 import sys
 
-from criteo_api_sdk.api.analytics_api import AnalyticsApi
-from criteo_api_sdk.models import StatisticsReportQueryMessage
-from criteo_api_sdk import Configuration, ApiClient
+from {{packageName}}.api.analytics_api import AnalyticsApi
+from {{packageName}}.models import StatisticsReportQueryMessage
+from {{packageName}} import Configuration, ApiClient
 
 if __name__ == '__main__':
     if len(sys.argv) != 3:
```

Comment:
It could be a good opportunity to fix the example completely. I've noticed that:
- Line 20, 21: start_date, end_date parameters expect datetime objects and not strings: this should do the trick: `datetime.strptime("2022-07-01", '%Y-%m-%d')`
- Line 27: method `get_adset_report_with_http_info` doesn't exist, method `get_adset_report` should be called instead, instead of asserting the status code, I guess `pprint` the response is enough